src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,107 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileReaderImplTest {
+    public static final String FILE_NAME = "src/main/java/core/basesyntax/resources/data.csv";
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+    private FileReader fileReader = new FileReaderImpl();
+
+    @Test
+    public void readFromFile_correctPath_Ok() {
+        Path file = Paths.get(FILE_NAME);
+        long count = 0;
+        List<String> arrayList = fileReader.readFromFile(FILE_NAME);
+        try {
+            count = Files.lines(file).count();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
e.printStackTrace(); is a bad practice, rather throw exception
this method is testing a normal situation if there would be a problem with the file - you should throw an exception